tests/unit/test_unit_telemetry.py

```diff
@@ -0,0 +1,18 @@
+import json
+
+import pytest
+
+from any_agent import AgentFramework
+from any_agent.telemetry import TelemetryProcessor
+
+
+@pytest.mark.parametrize("framework", list(AgentFramework))
```

Comment:
There is a fixture already for the `AgentFramework` called `agent_framework: AgentFramework`. I suggest using that instead of the parametrize in this test